selenium_webdriver/src/main/java/core/patterns/CustomDriverDecorator.java

```diff
@@ -0,0 +1,76 @@
+package core.patterns;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Reporter;
+
+import java.util.List;
+import java.util.Set;
+
+public class CustomDriverDecorator implements WebDriver {
```

Comment:
also, you may add here a custom method if you want

Reply:
Fixed